common/css/tab.css

```diff
@@ -214,14 +214,42 @@ button {
 .js-toc .toc-list {
   list-style-type: none;
   color: #43464b;
+  padding-top: 10px;
 }
 
-.js-toc > .js-toc-result > .toc-list {
-  padding: 10px 0 0;
+.js-toc .toc-list li {
+  margin-bottom: 6px;
+  padding: 7px;
+  font-size: 16px;
+  background-color: rgba(0,0,0,0.05);
+  transition: background-color 0.07s;
 }
 
-.js-toc .is-active-link {
-  color: black;
+.js-toc .toc-list li:hover { 
+  background-color: rgba(0,0,0,0.1);
+}
+
+.js-toc .toc-list li.is-active-li {
+  background-color: rgba(53,155,183,1.0);
+  color: white !important;
+}

```

Comment:
Tässä voisi vaihtaa taustavärin vaaleansiniseksi rgba(224, 244, 254, 0.9) ja color:in mustaksi, niin tulee tarpeeksi kontrasti. Päävalikolla on sama ongelma, mutta muutan sen, kun olet tämän tehnyt ensin.